src/pages/DataAccessRequestApplication_new.js

```diff
@@ -260,6 +268,26 @@ const DataAccessRequestApplicationNew = (props) => {
     }, 200));
   }, [forcedScroll]);
 
+  //Can't do uploads in parallel since endpoints are post and they both alter attributes in json column
+  //If done in parallel, updated attribute of one document will be overwritten by the outdated value on the other
```

Comment:
This comment/code was copied over from the original dar form. I'll look into it

Reply:
I agree with you that it seems there isn't a way to do this. Long term, though, do we want to use FileStorageObjects here? If so this whole code path will change anyway. I also personally think from a "pure REST" perspective, creating multiple resources from one POST call might be unexpected / not desired, since there aren't really any performance concerns that I know of.